4 - Tables and Persistence/1 - Protocols/lab/Lab - Protocols.playground/Pages/2. App Exercise - Printable Workouts.xcplaygroundpage/Contents.swift

```diff
@@ -39,5 +55,15 @@ class Workout {
  */
 import Foundation
 
+let jsonEncoder = JSONEncoder()
+
+do {
+    let jsonData = try jsonEncoder.encode(workoutOne) //takes humanOne object to retrieve data
+    let jsonString = String(data: jsonData, encoding: .utf8) //JSONEncoder will give us the JSPN data used to retrieve JSON string
+    print("JSON String : " + jsonString!)
+}
+catch {
+}
```

Comment:
@Natasha-A  in this scenario, think about what you would want to happen in your catch block if this was a real application

Reply:
Use catch block to display to user error message or response